src/aioswitcher/bridge.py

```
@@ -14,12 +14,33 @@
 
 """Switcher integration, UDP Bridge module."""
 
-from asyncio import BaseTransport, DatagramProtocol, get_running_loop
+import socket as socket_module
+from asyncio import (
+    AbstractEventLoop,
+    BaseTransport,
+    DatagramProtocol,
+    DatagramTransport,
+    Future,
+    TimeoutError,
```

Ruff (0.15.20)

[error] Import `TimeoutError` is shadowing a Python builtin

(A004)